create_aio_app/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.2'
+__version__ = '0.0.3'
```

Comment:
Hardcoding makes versions inconsistent with Git tags. Any git tag triggers deployment, for example. Even if it doesn't match, forgotten, or mistyped. I encourage using setuptools-scm plugin instead.

Reply:
👍